app/build.gradle

```diff
@@ -137,6 +138,21 @@ dependencies {
   testImplementation "org.assertj:assertj-core:3.11.1"
   testImplementation 'com.jraska.livedata:testing-ktx:1.1.0'
   testImplementation 'androidx.arch.core:core-testing:2.0.1'
+  // Leak canary
+  implementation 'com.squareup.leakcanary:leakcanary-android-no-op:1.5.4'
```

Comment:
this dependency already exists

Reply:
@iadeelzafar Can you make the changes suggested by Sean asap?

Reply:
Fixed a847783a61c8903dc4999b4532e3828b3462a5cb